apps/meteor/app/authorization/server/constant/permissions.ts

```diff
@@ -56,6 +56,7 @@ export const permissions = [
 	{ _id: 'manage-own-outgoing-integrations', roles: ['admin'] },
 	{ _id: 'manage-own-incoming-integrations', roles: ['admin'] },
 	{ _id: 'manage-oauth-apps', roles: ['admin'] },
+	{ _id: 'view-oauth-apps', roles: ['admin', 'owner', 'moderator', 'user'] },
```

Comment:
```suggestion
	{ _id: 'view-oauth-apps', roles: ['admin', 'user'] },
```
Since this new permission is meant for global scope(and not rooms scope), we should keep only roles that fit this scope